src/i18n/es/nav.ts

```diff
@@ -48,7 +48,7 @@ export default [
 		key: 'core-concepts/astro-syntax',
 	},
 	{
-		text: 'Frameworks de Interfaz de Usuario',
+		text: 'Frameworks UI',
```

Comment:
```suggestion
		text: 'Componentes de otros frameworks',
```
En los docs aparece de la siguiente manera y para que empate la navegación yo creo que debería ser puesto de la manera en la que te hago la sugerencia. ¿Qué opinas?
https://docs.astro.build/es/core-concepts/framework-components/

Reply:
Sería una buena idea, sin embargo, el problema está en que en el original no son iguales. En el original, el menú de navegación es "UI Frameworks" y "Framework Components", mientras que el título de la página es diferente. Aunque no sea 100% preciso, creo que debemos mantenerlo de esta manera para ser fieles a el nav original

Reply:
Entonces vamos a dejarlo así.

Reply:
Ok, marco esto como resuelto entonces